perma_web/perma/email.py

```diff
@@ -61,6 +61,21 @@ def send_admin_email(title, from_address, request, template="email/default.txt",
         headers={'Reply-To': from_address}
     ).send(fail_silently=False)
 
+def send_user_email_copy_admins(title, from_address, to_addresses, request, template="email/default.txt", context={}):
```

Comment:
Right, have made all suggested improvements except this one. Think I'd rather bite that off later, if you don't mind.